name-service/program/tests/functional.rs

```diff
@@ -175,6 +176,58 @@ async fn test_name_service() {
     .unwrap();
     println!("Name Record Header: {:?}", name_record_header);
 
+    let data = "hello".as_bytes().to_vec();
+    let update_instruction = update(
+        program_id,
+        space as u32,
+        data,
+        name_account_key,
+        sol_subdomains_class.pubkey(),
+        Some(name_record_header.parent_name),
+    )
+    .unwrap();
+
+    sign_send_instruction(
+        &mut ctx,
+        update_instruction.clone(),
+        vec![&sol_subdomains_class],
+    )
+    .await
+    .unwrap_err();
+
+    #[allow(clippy::integer_arithmetic)]
+    let new_space = space * 2;
```

Comment:
nit: you can just the checked version here 
```suggestion
    let new_space = space.checked_mul(2).unwrap();
```